docs/usage.rst

```diff
@@ -113,12 +125,49 @@ Bob is the new owner:
 
 .. code-block:: python
 
-    transfer_tx['transaction']['conditions'][0]['owners_after'][0] == bob.verifying_key
+    >>> transfer_tx['transaction']['conditions'][0]['owners_after'][0] == bob.verifying_key
     True
 
 Alice is the former owner:
 
 .. code-block:: python
 
-    transfer_tx['transaction']['fulfillments'][0]['owners_before'][0] == alice.verifying_key
+    >>> transfer_tx['transaction']['fulfillments'][0]['owners_before'][0] == alice.verifying_key
     True
+
+
+Transaction Status
+------------------
+Using the ``id`` of a transaction, its status can be obtained:
+
+.. code-block:: python
+
+    >>> bdb.transactions.status(creation_tx['id'])
+    {'status': 'valid'}
+
+Handling cases for which the transaction ``id`` may not be found:
+
+.. code-block:: python
+
+    import logging
+
+    from bigchaindb_driver import BigchainDB
+    from bigchaindb_driver.exceptions import NotFoundError
+
+    logger = logging.getLogger(__name__)
+    logging.basicConfig(format='%(asctime)-15s %(status)-3s %(message)s')
```

Comment:
Maybe just use `print` here to make it less complicated? If they want to use the logger, I'm sure they'll be able to figure out how elsewhere.